app/views/editor/post.html.erb

```diff
@@ -79,6 +79,10 @@
         <h3>Share your work</h3>
       <% end %>
   
+      <% if params[:redirect] == 'question' %>
+        <%= hidden_field_tag 'redirect', 'question' %>
```

Comment:
This is added to provide redirect to the question show path only when a `:redirect` parameter is given in the url. Well I thought of redirecting the questions to the notes show page if questions are posted from post a research note link so that the user doesn't gets redirected to the questions show page which is currently under construction.